LanguageApp/app/screens/LearnerScenarioScreen.js

```diff
@@ -91,10 +89,10 @@ function LearnerScenarioScreen(translatorId) {
   }
 
   return (
-    <Screen>
+    <View>
       <KeyboardAwareScrollView>
         <View style={styles.container}>
-          <AppTitle style={styles.text}>{scenario.title}</AppTitle>
+          {/* <AppTitle style={styles.text}>{scenario.title}</AppTitle> */}
```

Comment:
This can be removed since we are no longer using Title components, correct?

Reply:
yes!